README.md

```diff
@@ -1,5 +1,7 @@
 # LSA Program
 
+[Acces to roadmap here:](https://www.lucidchart.com/invitations/accept/61d72a6b-d843-42b5-b54a-22c7f85e84d3)
```

Comment:
Shall we change this to `Access to roadmap here: [roadmap](https://www.lucidchart.com/invitations/accept/61d72a6b-d843-42b5-b54a-22c7f85e84d3)`